### DISCUSSION TITLE
Does rspack have the InlineChunkHtmlPlugin？

### DISCUSSION THREAD

**Discussion by chenwll** [web-infra-dev/rspack#6647] [Q&A] [answered]
Does rspack have the InlineChunkHtmlPlugin？

**Comment by chenjiahan** [comment, marked as answer]
You can use `inline-chunk-html-plugin` with `hmlt-webpack-plugin` in Rspack.

And Rsbuild provides built-in supports for it: https://rsbuild.dev/config/output/inline-scripts
